api-report/telemetry-utils.api.md

```diff
@@ -309,9 +309,6 @@ export class ThresholdCounter {
 // @public
 export function wrapError<T extends LoggingError>(innerError: unknown, newErrorFn: (message: string) => T): T;
 
-// @public
-export function wrapErrorAndLog<T extends LoggingError>(innerError: unknown, newErrorFn: (message: string) => T, logger: ITelemetryLogger): T;
```

Comment:
adding a new standalone function is fine in main from an api perspective

Reply:
What about removing and existing one?

Reply:
wtf i'm so blind that's red not green
if it was only used internally within fluid it's fine, otherwise it's breaking (but removing anything is breaking by default generally)